src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/goto--options--.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'goto(url, [options])'
+excerpt: 'Browser module: page.goto(url, [options]) method'
+---
+
+Navigates to the specified url and returns the main resource response.
+
+Navigating to `about:blank` or navigation to the same URL with a different hash, will succeed and return `null`.
+
+<TableWithNestedRows>
+
+| Parameter       | Type   | Default | Description                                                                                                                                                                                                                           |
+|-----------------|--------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| url            | string  | `''`    |  URL to navigate page to. The url should include scheme, e.g. `https://`.                                                                                                               |
+| options         | object | `null`  |                                                                                                                                                                                                                      |
+| options.referer  | string| `''`  | Referer header value.                                                             |
+| options.timeout | number | `30000` | Maximum operation time in milliseconds. Pass `0` to disable the timeout. The default value can be changed via the [browserContext.setDefaultNavigationTimeout(timeout)](/javascript-api/k6-experimental/browser/browsercontext/setdefaultnavigationtimeout/), [browserContext.setDefaultTimeout(timeout)](/javascript-api/k6-experimental/browser/browsercontext/setdefaulttimeout/), [page.setDefaultNavigationTimeout(timeout)](/javascript-api/k6-experimental/browser/page/setdefaultnavigationtimeout/) or [page.setDefaultTimeout(timeout)](/javascript-api/k6-experimental/browser/page/setdefaulttimeout/) methods. Setting the value to `0` will disable the timeout. |
+| options.waitUntil | string | `load` | When to consider operation to have succeeded. See [Events](#events) for more details. |
+
+</TableWithNestedRows>
+
```

Comment:
We should probably have a _Returns_ sub heading in here.

Reply:
Resolve in https://github.com/grafana/k6-docs/commit/95398b184f9afe52df89593693d16dfa2d7085c5